deploy/FlapperDeploy.sol

```diff
@@ -24,6 +24,7 @@ import { FlapperUniV2 } from "src/FlapperUniV2.sol";
 import { FlapperUniV2SwapOnly } from "src/FlapperUniV2SwapOnly.sol";
 import { FlapperMom } from "src/FlapperMom.sol";
 import { OracleWrapper } from "src/OracleWrapper.sol";
+import { Splitter } from "src/Splitter.sol";
```

Comment:
Probably need to mention the Splitter in the README as well.

Reply:
Added